src/plugins/intel_gpu/src/kernel_selector/cl_kernels/fully_connected_gpu_bf_tiled.cl

```diff
@@ -17,6 +17,47 @@
 // DISPATCH_FSV - output coordinates for each sub-group are calculated from linearized coordinates
 // DISPATCH_BSV   as if they laid in bs_fs_bsv_fsv format, these macros describe fsv and bsv factors;
 
+#define INPUT_LOAD_SIZE                     4
+
+#if FC_KERNEL_DYNAMIC_QUANTIZE
+KERNEL(quantize_input)(
```

Comment:
Hmm it is implemented this way because of different code path is required between 1st token(dyn quan is applied) and 2nd token(dyn quan is not applied). Let's discuss offline how we can improve the code.